Source/ModelGroup.cs

```diff
@@ -17,18 +17,26 @@ internal abstract class ModelGroup
         public int Id = -1;
         public bool NoSampleImages = false;
 
-        protected Runtime.Image GetImage(List<string> imageList, string name)
+        protected Runtime.Image GetImage(List<string> imageList, string name, bool isFigure = false)
```

Comment:
rename to UseImage

Reply:
also make this private and create two functions UseTexture and UseFigure.

Reply:
Fixed